developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md

```diff
@@ -69,15 +69,11 @@ This document describes how to configure your public FullNode using both the met
 
 ### Using Aptos-core source code
 
-1. Fork and clone the Aptos repo.
+1. Clone the Aptos repo.
 
-    - Fork the Aptos Core repo by clicking on the **Fork** on the top right of this repo page: https://github.com/aptos-labs/aptos-core.
-    - Clone your fork.
-
-      ```
-      git clone https://github.com/<YOUR-GITHUB-USERID>/aptos-core
-
-      ```
+    ```
+    git clone https://github.com/aptos-labs/aptos-core
```

Comment:
You need .git at the end of all of these right?

Reply:
yes, I was going to get to it this morning. We do need .git but I tested without it and it works. However, I will document it with the .git.